src/algebra/quandle.lean

```diff
@@ -585,26 +585,38 @@ end to_envel_group.map_aux
 /--
 Given a map from a rack to a group, lift it to being a map from the enveloping group.
```

Comment:
Nice catch! I didn't realize the adjunction could be written as an equivalence like this.  I'd update the docstring to make note of it:
```
Given a map from a rack to a group, lift it to being a map from the enveloping group.
More precisely, the `envel_group` functor is left adjoint to `quandle.conj`.
```

The lemmas `to_envel_group.univ` and `to_envel_group.univ_uniq` seem superfluous now, though maybe they're useful as reminders of the universal property.

Reply:
The module documentation for this, under "Main statements`, should probably be replaced by something like
```
* `rack.envel_group` is left adjoint to `quandle.conj` (`to_envel_group.map`).
  The universality statements are `to_envel_group.univ` and `to_envel_group.univ_uniq`.
```

Reply:
> though maybe they're useful as reminders of the universal property.

I'd prefer to keep them until we work out a better way to bundle universal properties - they're a nice marker for "there is a universal property lurking here"